.changeset/itchy-kiwis-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': minor
```

Comment:
I think we should aim to keep this a non-breaking change, which would make this a `patch` bump as we're on 0.x